### PR TITLE
FM-25 Configure MaterialUI

### PR DESCRIPTION
**Pull request by GR33NBR1CK**
# Description

I added configuration for MaterialUI and added a test component 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Added test component.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] The related memme has been added
me and the project

### CODE REVIEW

**Review by MaciejMogilewski — APPROVED** [review]
I'm no expert, but it seems ok

File: flashcards/web/package.json [hunk 1]
```diff
@@ -15,19 +15,25 @@
         "watch": "vitest watch"
     },
     "dependencies": {
+        "@emotion/react": "^11.10.6",
+        "@emotion/styled": "^11.10.6",
+        "@fontsource/roboto": "^4.5.8",
+        "@mui/material": "^5.11.15",
+        "@mui/styled-engine-sc": "^5.11.11",
         "@tanstack/react-query": "^4.26.1",
```
Comment (MWasile) [inline flashcards/web/package.json:23]: i think u should also add @mui/icons-material

**Review by MWasile — CHANGES_REQUESTED** [review]
if you remove package-lock.json, Dockerfile will stop working
`COPY ./package.json ./package-lock.json ./`

File: flashcards/web/package.json (hunk 1 above)
Comment (GR33NBR1CK) [reply flashcards/web/package.json:24]: done :)

**Review by MWasile — APPROVED** [review]
good 👍

**Pull request merged**